### ISSUE TITLE
redirect to use or extend redirect response

### ISSUE THREAD

**Issue by mybigman** [starlite-api/starlite#278]
Currently Redirect does not return a response so its not usable like Starlettes `RedirectResponse`.

Is there another approach or just use Starlettes?

```python
@post()
async def post_login(
    self,
    data: UserLogin = Body(media_type=RequestEncodingType.URL_ENCODED),
) -> Response:
    response: Response = RedirectResponse(url="/somewhere", status_code=303)
    response.set_cookie(...)

    return response
```

**Comment by Goldziher** [comment]
You can use the `after_request` hook to modify the response Starlite creates.

**Comment by mybigman** [comment]
Yes I know that, however I am doing stuff in the handler first before passing values into the cookie.

**Comment by Goldziher** [comment]
So what's the problem exactly? You can do what you do and return a redirect response as is. What's the issue with this setup?

**Comment by mybigman** [comment]
The is no issue per say. Just a nice to have for consistency.

```python
from starlite import Controller, HTTPException, RedirectResponse, get, post

# rather than
from starlette.responses import RedirectResponse
from starlite import Controller, HTTPException, get, post
```

I guess I am use to `fastapi` where mostly everything comes from the same framework rather then having to remember this comes from the underlying framework.

Sorry if it wasn't clear feel free to close.

**Comment by Goldziher** [comment]
> The is no issue per say. Just a nice to have for consistency.
> 
> ```python
> from starlite import Controller, HTTPException, RedirectResponse, get, post
> 
> # rather than
> from starlette.responses import RedirectResponse
> from starlite import Controller, HTTPException, get, post
> ```
> 
> I guess I am use to `fastapi` where mostly everything comes from the same framework rather then having to remember this comes from the underlying framework.
> 
> Sorry if it wasn't clear feel free to close.

I see, you basically want re-exports. I'd suggest you switch to an IDE that handles this for you.

**Comment by Goldziher** [comment]
See #338 I opened above
